scripts/build.ts

```diff
@@ -231,7 +231,7 @@ class Builder {
 
 			const [productJson, packageJson] = await Promise.all([
 				merge("product", { commit, date }),
-				merge("package", { codeServerVersion: `${codeServerVersion}-vsc${vscodeVersion}` }),
+				merge("package", { codeServerVersion: codeServerVersion }),
```

Comment:
Not a big deal but this could be `{ codeServerVersion }`.

Reply:
nice catch